triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/dao/DbTriggerDao.scala

```diff
@@ -38,6 +39,15 @@ abstract class DbTriggerDao protected (
   protected implicit def uuidPut: Put[UUID]
   protected implicit def uuidGet: Get[UUID]
 
+  implicit val readAsPut: Put[Set[Party]] =
+    implicitly[Put[String]].contramap { (readAs: Set[Party]) =>
+      readAs.map(Tag.unwrap).mkString(",")
+    }
+
+  implicit val readAsGet: Get[Set[Party]] = implicitly[Get[String]].map {
+    _.split(",").map(_.trim).toSet.filter(_.nonEmpty).map(Party.apply)
```

Comment:
Using `%` instead of `,` might be safer as it is explicitly reserved from being an identifier character by the LF spec.

Reply:
didn't know about that, thanks!

Reply:
Another note, `PartyIdString` allows spaces. So `trim` is not OK here.